javascript/net/grpc/web/Makefile

```diff
@@ -16,14 +16,15 @@ CXX ?= g++
 CPPFLAGS += -I/usr/local/include -pthread
 CXXFLAGS += -std=c++11
 LDFLAGS += -L/usr/local/lib -lprotoc -lprotobuf -lpthread -ldl
+PREFIX ?= /usr/local
 
 all: protoc-gen-grpc-web
 
 protoc-gen-grpc-web: grpc_generator.o
 	$(CXX) $^ $(LDFLAGS) -o $@
 
 install: protoc-gen-grpc-web
-	install protoc-gen-grpc-web /usr/local/bin/protoc-gen-grpc-web
+	install -Dm755 protoc-gen-grpc-web $(PREFIX)/bin/protoc-gen-grpc-web
```

Comment:
I don't think `-Dm755` work on macOS (BSD) version of `install`..

Is there a way you can make this work for both? :)

Reply:
Can we leave this `-Dm755` out and just use the system default? the `PREFIX` change looks good.

Reply:
I can either drop it or add a `mkdir -p $(PREFIX)` above `install`

Reply:
Yeah this is cool with me.. But i'll defer to @stanley-cheung's for approval :)

Thanks for the change :)